storage/mysql/mysql.go

```diff
@@ -252,20 +253,37 @@ func (s *Storage) ReadTile(ctx context.Context, level, index, width uint64) ([]b
 	var tile []byte
 	if err := row.Scan(&tile); err != nil {
 		if err == sql.ErrNoRows {
-			return nil, nil
+			return nil, os.ErrNotExist
 		}
 
 		return nil, fmt.Errorf("scan tile: %v", err)
 	}
 
-	// Return nil when returning a partial tile on a full tile request.
-	if width == 256 && uint64(len(tile)/32) != width {
-		return nil, nil
+	minSize := partialTileSize(level, index, minTreeSize)
```

Comment:
The partial tile size calculated from the minimum tree size seems to be the same as the `width` from the URL. Shall we rename `minSize` back to `width`?

Reply:
Renamed as agreed 1:1